### PR TITLE
Add citation of the library

### LINKED ISSUE #216: Add citation of the library
**Is your feature request related to a problem? Please describe.**
One may want to cite the library, provide a citation.

**Describe the solution you'd like**
```latex
 @misc{kastryulin_zakirov_prokopenko_2019,
 title={PyTorch Image Quality - Metrics and Measure for Image Quality Assessment}, url={https://github.com/photosynthesis-team/piq}, author={Kastryulin, Sergey and Zakirov, Djamil and Prokopenko, Denis},
 year={2019},
 month={Dec}} 
```

**Additional context**
The format is discussable, the citation itself can be added to a separate CITATION file like [it is done in PyTorch](https://github.com/pytorch/pytorch/blob/master/CITATION) or directly to the `README.md` file.

### PR DESCRIPTION
**Pull request by snk4tr**
Closes #216 

## Proposed Changes

  - Add citation of the library

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,6 +195,23 @@ We appreciate all your contributions. If you plan to:
 Please see the [contribution guide](CONTRIBUTING.md) for more information.
 
 
+<!-- CITATION -->
+### Citation
+
```
Comment (denproc) [inline README.md:200]: If you use PIQ in your project, please, cite it as follows.

File: README.md [hunk 2]
```diff
@@ -195,6 +195,23 @@ We appreciate all your contributions. If you plan to:
...
+@misc{piq,
+  title={{PyTorch Image Quality}: Metrics and Measure for Image Quality Assessment},
+  url={https://github.com/photosynthesis-team/piq},
+  note={Open source software available from https://github.com/photosynthesis-team/piq},
+  author={
+    Sergey Kastryulin and
+    Djamil Zakirov and
+    Denis Prokopenko},
+  year={2019},
+}
```
Comment (denproc) [inline README.md:211]: DOI: 10.5281/zenodo.4478655

Comment (denproc) [reply README.md:211]: New release will trigger Zenodo to update the link.

**Review by denproc — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -195,6 +195,23 @@ We appreciate all your contributions. If you plan to:
...
 
 
+<!-- CITATION -->
+### Citation
+
+```tex
+@misc{piq,
+  title={{PyTorch Image Quality}: Metrics and Measure for Image Quality Assessment},
+  url={https://github.com/photosynthesis-team/piq},
+  note={Open source software available from https://github.com/photosynthesis-team/piq},
```
Comment (denproc) [inline README.md:205]:
```suggestion
  note={Open source software available at https://github.com/photosynthesis-team/piq},
```

**Comment by snk4tr** [comment]
@denproc done

File: README.md (hunk 1 above)
Comment (snk4tr) [reply README.md:200]: Added this

**Review by denproc — APPROVED** [review]

**Pull request merged**